frontend/amundsen_application/static/js/components/Table/index.tsx

```diff
@@ -37,12 +39,14 @@ export interface TableOptions {
   isLoading?: boolean;
   numLoadingBlocks?: number;
   rowHeight?: number;
-  preExpandRow?: number;
-  expandRow?: (rowValue: any, index: number) => React.ReactNode;
-  onExpand?: (rowValues: any, index: number) => void;
-  onCollapse?: (rowValues: any, index: number) => void;
+  preExpandPanelKey?: string;
```

Comment:
We should have comments on all this props.
You can copy/paste most of them from https://github.com/lyft/legoland/blob/master/packages/dsml-ui/table/src/Table.tsx#L38